insalan/tournament/views/bracket.py

```diff
@@ -0,0 +1,46 @@
+from django.utils.translation import gettext_lazy as _
+from django.core.exceptions import PermissionDenied
+
+from rest_framework import generics, permissions, status
+from rest_framework.response import Response
+from rest_framework.exceptions import NotFound
+
+import insalan.tournament.serializers as serializers
+
+from ..models import Bracket, KnockoutMatch, MatchStatus, BracketSet, validate_match_data
+# , InvalidScores, InvalidTeamList, InvalidTeamScore, NotOngoingMatch
+from ..manage import update_match_score, update_next_knockout_match
+
+from .permissions import ReadOnly, Patch
+
+class BracketMatchScore(generics.UpdateAPIView):
+    """Update score of a bracket match"""
+
+    queryset = KnockoutMatch.objects.all().order_by("id")
+    serializer_class = serializers.KnockoutMatchSerializer
+    permission_classes = [permissions.IsAuthenticated]
+
+    def patch(self, request, *args, **kwargs):
+        user = request.user
+        data = request.data
+
+        try:
+            match = KnockoutMatch.objects.get(pk=kwargs["match_id"],bracket=kwargs["bracket_id"])
```

Comment:
similar concern here, check the keys prior and raise bad request if one of them is missing or the wrong type